src/enums/relationship-types.ts

```diff
@@ -0,0 +1,7 @@
+export enum RelationshipTypes {
+    COURT_ORDERED_PARTY = 'Court Ordered Party',
+    DIRECTOR = 'Director',
+    HEIR_LEGAL_REP = 'Heir or Legal Representative',
+    OFFICER = 'Officer',
+    SHAREHOLDER = 'Shareholder',
+  }
```

Comment:
Looks good!

Please also update the index file in the enums folder, as you will be publishing an updated version of the enums package as part of this PR.